docs/src/index.md

```diff
@@ -1,8 +1,36 @@
-# GPLikelihoods.jl
+```@meta
+CurrentModule = GPLikelihoods
+```
+# GPLikelihoods
+
+[`GPLikelihoods.jl`](https://github.com/JuliaGaussianProcesses/GPLikelihoods.jl) provides a practical interface to connect non-conjugate likelihoods
+with Gaussian Processes.
+The API is very basic: Every `AbstractLikelihood` object is a [functor](https://docs.julialang.org/en/v1/manual/methods/#Function-like-objects-1)
+that takes the output of a Gaussian process as input and returns a 
```

Comment:
Could we be more specific about what we mean by "output". Do we really just mean `Real`s by "output"?

Reply:
I see your point, I tried to be general since we accept both `Real` and `Vector` but this might be more confusing than anything. I will try to improve this part